remotespark/livyclientlib/livysession.py

```diff
@@ -12,7 +13,8 @@
 from .livyreliablehttpclient import LivyReliableHttpClient
 
 
-class LivySession(object):
+
+class LivySession(GuidMixin, object):
```

Comment:
Don't you have to call the parent's constructor `__init__` method for the mixin to work?